apps/passport-server/test/user/testDeviceLogin.ts

```diff
@@ -1,63 +1,67 @@
-import { User } from "@pcd/passport-interface";
+import {
+  PCDpassUserJson,
+  requestDeviceLogin,
+  requestUser,
+  User
+} from "@pcd/passport-interface";
 import { Identity } from "@semaphore-protocol/identity";
 import { expect } from "chai";
-import httpMocks from "node-mocks-http";
 import { PCDpass } from "../../src/types";
 
 export async function testDeviceLogin(
   application: PCDpass,
   email: string,
   secret: string
 ): Promise<{ user: User; identity: Identity } | undefined> {
-  const { userService } = application.services;
   const identity = new Identity();
   const commitment = identity.commitment.toString();
 
-  const newDeviceLoginResponse = httpMocks.createResponse();
-  await userService.handleNewDeviceLogin(
-    secret,
+  const deviceLoginResponse = await requestDeviceLogin(
+    application.expressContext.localEndpoint,
     email,
-    commitment,
-    newDeviceLoginResponse
+    secret,
+    commitment
   );
```

Comment:
I've replaced all the intances where we call or pretend to call the backend api in our tests with the implementations that live in `@pcd/passport-interface`